### DISCUSSION TITLE
[Devtool] An error occurred in hook getInspectorTree registered by plugin org.vuejs.vue2-internal

### DISCUSSION THREAD

**Discussion by mashirozx** [vuejs/pinia#907] [Help and Questions]
### Reproduction

https://git.moezx.cc/mashiro/agk-common/src/commit/a83ee7467596978d5d7d41e38b4bd5837d5f6598

Reproduction on commit tree `a83ee7467596978d5d7d41e38b4bd5837d5f6598` 

### Steps to reproduce the behavior
```
    "pinia": "^2.0.6",
    "vue": "^2.6.14",
```

Devtools: 6.0.0 beta 20

`useTagsStore` in a layout component [code here](https://git.moezx.cc/mashiro/agk-common/src/commit/a83ee7467596978d5d7d41e38b4bd5837d5f6598/src/layouts/components/Tabs.vue#L16)

### Expected behavior

No error or warnings in browser console

### Actual behavior

![image](https://user-images.githubusercontent.com/16148054/146484138-0da22b18-5916-43f3-b128-190f0ef9b3ac.png)

### Additional information

I tried to use store with a setTimeout for 200ms, then it works without error, but this is not what I expected.

![image](https://user-images.githubusercontent.com/16148054/146485046-e4f7d88a-f977-40d0-88ae-bfbd63d71e78.png)

Seems there is issue for DevTools to detect Vue app instance when initializing the store.

**Comment by posva** [comment]
As stated in the issue template I need a boiled down repro. Let me know if you find one. Moving this to a discussion.

**Comment by nicholasklaene** [comment]
I am having the same issue using the chrome browser extension.

**Reply by nicholasklaene** [reply]
All functionality seems to be working, just an annoying console error

**Reply by hosseinghs** [reply]
I have the same issue in chrome , but I see no errors in firefox

**Comment by Dawn-ferry** [comment]
This should be Vue.js devtools 6.0.0 beta 21 fault，Waiting for repair or close  devtools's beta

**Reply by lol2x** [reply]
vue devtools out of beta and error is still there

**Reply by abugi** [reply]
I had the same issue.
I turned off "Custom Inspectors" in Vue devTools global settings and the error disappeared.

**Reply by justice47** [reply]
@abugi thanks, fixed for me

**Reply by zoyuxb** [reply]
Or you may downgrade to the old version https://devtools.vuejs.org/guide/installation.html#legacy

**Reply by hosseinghs** [reply]
> I had the same issue. I turned off "Custom Inspectors" in Vue devTools global settings and the error disappeared.

how can I turn it off too ?

**Reply by abugi** [reply]
In your Vue devTools...

- click on the 3 dots at the top right hand corner to reveal the options
- select "Devtools plugin" option
- click on "Vue 2" in the left side bar
- click the switch to turn off custom inspectors

PS: I turned it back on later and the error did not come back.

**Reply by tom-evaristo** [reply]
> In your Vue devTools...
> 
>     * click on the 3 dots at the top right hand corner to reveal the options
> 
>     * select "Devtools plugin" option
> 
>     * click on "Vue 2" in the left side bar
> 
>     * click the switch to turn off custom inspectors
> 
> 
> PS: I turned it back on later and the error did not come back.

This worked for me to. using laravel-mix

current setup is:

```   
 "devDependencies": {
        "autoprefixer": "^10.4.7",
        "axios": "^0.19",
        "cross-env": "^7.0",
        "laravel-mix": "^6.0.49",
        "lodash": "^4.17.13",
        "pinia": "^2.0.14",
        "postcss": "^8.4.14",
        "resolve-url-loader": "^3.1.0",
        "sass": "^1.15.2",
        "sass-loader": "^8.0.0",
        "tailwindcss": "^3.1.6",
        "vue": "^3.2.37",
        "vue-loader": "^16.8.3"
    }
```

Disabled and enabled the Custom Inspector and the error went away. weird

**Reply by natsuki-engr** [reply]
In my case, i turned off custom inspectors of pinia, and turned on immediately. the error didn't come back.
